### PR TITLE
QoL: Cleaning Proto Files

### PR DESCRIPTION
**Pull request by TheMarstonConnell**
Currently the protos are a mess. Some generated with Ignite before ignite handled data types that weren't strings, some generated by real humans who didn't have a "proto-standardization manual" in front of them. 

From here on out we are adopting a protobuf styling standard following that of the Cosmos-SDK. Types will be clearly defined in Camel Case to match golang types, but fields in those types will be formatted with snake case to make them distinguishable from each other at a glance.

### CODE REVIEW

File: proto/canine_chain/filetree/tx.proto [hunk 1]
```diff
@@ -83,48 +83,48 @@ message MsgMakeRootV2 {
...
 }
 
 message MsgResetViewersResponse {}
 
 message MsgChangeOwner {
   string creator = 1;
   string address = 2;
-  string fileOwner = 3;
+  string file_owner = 3;
   string newOwner = 4;
```
Comment (karnthis) [inline proto/canine_chain/filetree/tx.proto:128]: still camelCase

File: proto/canine_chain/filetree/tx.proto [hunk 2]
```diff
@@ -23,58 +23,58 @@ service Msg {
...
-  string viewerIds = 2;
+  string viewer_ids = 2;
   string address = 3;
-  string fileowner = 4;
+  string file_owner = 4;
 }
 
 message MsgRemoveViewersResponse {}
 
 message MsgMakeRoot {
```
Comment (karnthis) [inline proto/canine_chain/filetree/tx.proto:70]: MsgMakeRoot should be deprecated

File: proto/canine_chain/rns/query.proto [hunk 3]
```diff
@@ -16,100 +16,87 @@ option go_package = "github.com/jackalLabs/canine-chain/x/rns/types";
...
 message QueryParamsResponse {
   // params holds all the parameters of this module.
   Params params = 1 [ (gogoproto.nullable) = false ];
 }
 
-message QueryWhoisRequest { string index = 1; }
+message QueryName { string index = 1; }
 
-message QueryWhoisResponse { Whois whois = 1 [ (gogoproto.nullable) = false ]; }
+message QueryNameResponse { Names name = 1 [ (gogoproto.nullable) = false ]; }
```
Comment (karnthis) [inline proto/canine_chain/rns/query.proto:84]: should this be Names name or Name name?

File: proto/canine_chain/rns/query.proto [hunk 4]
```diff
@@ -118,26 +105,26 @@ message QueryAllBidsResponse {
...
 
-message QueryAllForsalesRequest {
+message QueryAllForSale {
   cosmos.base.query.v1beta1.PageRequest pagination = 1;
 }
 
-message QueryAllForsalesResponse {
-  repeated Forsale forsale = 1 [ (gogoproto.nullable) = false ];
+message QueryAllForSaleResponse {
+  repeated Forsale for_sale = 1 [ (gogoproto.nullable) = false ];
```
Comment (karnthis) [inline proto/canine_chain/rns/query.proto:119]: repeated Forsale or repeated ForSale?

File: proto/canine_chain/rns/tx.proto [hunk 5]
```diff
@@ -11,7 +11,7 @@ service Msg {
   rpc CancelBid(MsgCancelBid) returns (MsgCancelBidResponse);
   rpc List(MsgList) returns (MsgListResponse);
   rpc Buy(MsgBuy) returns (MsgBuyResponse);
-  rpc Delist(MsgDelist) returns (MsgDelistResponse);
+  rpc DeList(MsgDeList) returns (MsgDeListResponse);
```
Comment (karnthis) [inline proto/canine_chain/rns/tx.proto:14]: Delist is a single word and should not be PascalCase

File: proto/canine_chain/storage/query.proto [hunk 6]
```diff
@@ -16,153 +16,153 @@ option go_package = "github.com/jackalLabs/canine-chain/x/storage/types";
...
+  // Queries a list of FreeSpace items.
+  rpc FreeSpace(QueryFreeSpace) returns (QueryFreeSpaceResponse) {
     option (google.api.http).get =
-        "/jackal-dao/canine-chain/storage/freespace/{address}";
+        "/jackal/canine-chain/storage/free_space/{address}";
   }
 
-  // Queries a list of Freespace items.
-  rpc StoreCount(QueryStoreCountRequest) returns (QueryStoreCountResponse) {
+  // Queries a list of FreeSpace items.
```
Comment (karnthis) [inline proto/canine_chain/storage/query.proto:92]: // Queries a list of StoreCount items.

File: proto/canine_chain/storage/query.proto [hunk 7]
```diff
@@ -16,153 +16,153 @@ option go_package = "github.com/jackalLabs/canine-chain/x/storage/types";
...
 
   // Queries whether user can upload a file based on size
-  rpc FileUploadCheck(QueryFileUploadCheckRequest) returns (QueryFileUploadCheckResponse) {
+  rpc FileUploadCheck(QueryFileUploadCheck) returns (QueryFileUploadCheckResponse) {
     option (google.api.http).get =
-        "/jackal-dao/canine-chain/storage/file_upload_check";
+        "/jackal/canine-chain/storage/file_upload_check";
   }
 
   // Queries whether user can upload a file based on size
```
Comment (karnthis) [inline proto/canine_chain/storage/query.proto:136]: update comment

File: proto/canine_chain/storage/query.proto [hunk 8]
```diff
@@ -16,153 +16,153 @@ option go_package = "github.com/jackalLabs/canine-chain/x/storage/types";
...
-  rpc PriceCheck(QueryPriceCheckRequest) returns (QueryPriceCheckResponse) {
+  rpc PriceCheck(QueryPriceCheck) returns (QueryPriceCheckResponse) {
     option (google.api.http).get =
-        "/jackal-dao/canine-chain/storage/price_check/{duration}/{bytes}";
+        "/jackal/canine-chain/storage/price_check/{duration}/{bytes}";
   }
 
   // Queries whether user can upload a file based on size
```
Comment (karnthis) [inline proto/canine_chain/storage/query.proto:142]: update comment

File: proto/canine_chain/storage/query.proto [hunk 9]
```diff
@@ -209,13 +207,13 @@ message QueryAllFilesResponse {
   cosmos.base.query.v1beta1.PageResponse pagination = 2;
 }
 
-message QueryProviderRequest { string address = 1; }
+message QueryProvider { string address = 1; }
 
 message QueryProviderResponse {
-  Providers providers = 1 [ (gogoproto.nullable) = false ];
+  Providers provider = 1 [ (gogoproto.nullable) = false ];
```
Comment (karnthis) [inline proto/canine_chain/storage/query.proto:213]: Providers provider or Provider provider?

File: proto/canine_chain/storage/query.proto [hunk 10]
```diff
@@ -257,48 +255,48 @@ message QueryReportResponse {
...
-message QueryFreespaceRequest { string address = 1; }
+message QueryFreeSpace { string address = 1; }
 
-message QueryStoreCountRequest { string address = 1; }
+message QueryStoreCount { string address = 1; }
 
-message QueryFreespaceResponse { string space = 1; }
+message QueryFreeSpaceResponse { string space = 1; }
 
 message QueryStoreCountResponse { string count = 1; }
```
Comment (karnthis) [inline proto/canine_chain/storage/query.proto:273]: string or int?

File: proto/canine_chain/storage/query.proto [hunk 11]
```diff
@@ -257,48 +255,48 @@ message QueryReportResponse {
...
 }
 
-message QueryFreespaceRequest { string address = 1; }
+message QueryFreeSpace { string address = 1; }
 
-message QueryStoreCountRequest { string address = 1; }
+message QueryStoreCount { string address = 1; }
 
-message QueryFreespaceResponse { string space = 1; }
+message QueryFreeSpaceResponse { string space = 1; }
```
Comment (karnthis) [inline proto/canine_chain/storage/query.proto:271]: string or int?

File: proto/canine_chain/storage/query.proto [hunk 12]
```diff
@@ -316,7 +314,7 @@ message QueryFileUploadCheckResponse {
   bool valid = 1;
 }
 
-message QueryPriceCheckRequest {
+message QueryPriceCheck {
   string duration = 1;
```
Comment (karnthis) [inline proto/canine_chain/storage/query.proto:318]: string or int?

File: proto/canine_chain/storage/tx.proto [hunk 13]
```diff
@@ -83,12 +83,12 @@ message MsgSetProviderKeybase {
 
 message MsgSetProviderKeybaseResponse {}
 
-message MsgSetProviderTotalspace {
+message MsgSetProviderTotalSpace {
   string creator = 1;
   string space = 2;
```
Comment (karnthis) [inline proto/canine_chain/storage/tx.proto:88]: string or int?

File: proto/canine_chain/storage/tx.proto [hunk 14]
```diff
@@ -109,7 +109,7 @@ message MsgInitProvider {
   string creator = 1;
   string ip = 2;
   string keybase = 3;
-  string totalspace = 4;
+  string total_space = 4;
```
Comment (karnthis) [inline proto/canine_chain/storage/tx.proto:112]: string or int?

**Review by karnthis — COMMENTED** [review]
noticed a number of small inconsistencies, not sure if they are left for compatibility.

File: proto/canine_chain/rns/query.proto (hunk 3 above)
Comment (TheMarstonConnell) [reply proto/canine_chain/rns/query.proto:84]: unfortunately this one related to the datatype which I am unsure if I can change without breaking everything

File: proto/canine_chain/rns/query.proto (hunk 4 above)
Comment (TheMarstonConnell) [reply proto/canine_chain/rns/query.proto:119]: Same issue as names above

File: proto/canine_chain/filetree/query.proto [hunk 15]
```diff
@@ -13,50 +13,50 @@ option go_package = "github.com/jackalLabs/canine-chain/x/filetree/types";
...
     option (google.api.http).get =
-        "/jackal-dao/canine-chain/filetree/pubkeys/{address}";
+        "/jackal/canine-chain/filetree/pubkeys/{address}";
   }
 
   // Queries a list of Pubkey items.
-  rpc PubkeyAll(QueryAllPubkeysRequest) returns (QueryAllPubkeysResponse) {
-    option (google.api.http).get = "/jackal-dao/canine-chain/filetree/pubkeys";
+  rpc AllPubKeys(QueryAllPubKeys) returns (QueryAllPubKeysResponse) {
+    option (google.api.http).get = "/jackal/canine-chain/filetree/pubkeys";
```
Comment (karnthis) [inline proto/canine_chain/filetree/query.proto:39]: should be "/jackal/canine-chain/filetree/pub_keys" right?

File: proto/canine_chain/filetree/tx.proto [hunk 16]
```diff
@@ -23,109 +22,100 @@ service Msg {
...
-  string rootHashPath = 3;
-  string contents = 4;
-  string editors = 5;
-  string viewers = 6;
-  string trackingNumber = 7;
-}
 
 message MsgMakeRootResponse {}
 
 message MsgMakeRootV2 {
```
Comment (karnthis) [inline proto/canine_chain/filetree/tx.proto]: remove MsgMakeRoot, MsgMakeRootV2, and MsgMakeRootResponse. replace with discussed MsgProvisionFileTree and MsgProvisionFileTreeResponse

File: proto/canine_chain/rns/query.proto [hunk 17]
```diff
@@ -16,100 +16,87 @@ option go_package = "github.com/jackalLabs/canine-chain/x/rns/types";
...
+message QueryParams {}
 
 // QueryParamsResponse is response type for the Query/Params RPC method.
 message QueryParamsResponse {
   // params holds all the parameters of this module.
   Params params = 1 [ (gogoproto.nullable) = false ];
 }
 
-message QueryWhoisRequest { string index = 1; }
+message QueryName { string index = 1; }
```
Comment (karnthis) [inline proto/canine_chain/rns/query.proto:82]: should be int not string

File: proto/canine_chain/rns/query.proto [hunk 18]
```diff
@@ -16,100 +16,87 @@ option go_package = "github.com/jackalLabs/canine-chain/x/rns/types";
...
 }
 
 message QueryAllNamesResponse {
-  repeated Names names = 1 [ (gogoproto.nullable) = false ];
+  repeated Names name = 1 [ (gogoproto.nullable) = false ];
   cosmos.base.query.v1beta1.PageResponse pagination = 2;
 }
 
-message QueryBidRequest { string index = 1; }
+message QueryBid { string index = 1; }
```
Comment (karnthis) [inline proto/canine_chain/rns/query.proto:95]: should be int not string

**Review by karnthis — CHANGES_REQUESTED** [review]
looking good, down to the last few items!

File: proto/canine_chain/filetree/tx.proto (hunk 16 above)
Comment (karnthis) [reply proto/canine_chain/filetree/tx.proto]: resolved

File: proto/canine_chain/rns/query.proto (hunk 17 above)
Comment (TheMarstonConnell) [reply proto/canine_chain/rns/query.proto]: should be string ;P shouldn't be called index since it's not an index. Leftovers from ignite :/

**Review by karnthis — APPROVED** [review]
lgtm. there are comments that we can clean up of time

**Pull request merged**